waggle-dance-core/pom.xml

```diff
@@ -253,7 +254,7 @@
     <dependency>
       <groupId>com.hotels</groupId>
       <artifactId>beeju</artifactId>
-      <version>1.0.0</version>
+      <version>${beeju.version}</version>
```

Comment:
can you move BeeJU to the dependency management section in the parent POM and remove the version from this and the other module?